iocage_cli/snaplist.py

```diff
@@ -43,9 +43,14 @@ def cli(header, jail, _long, _sort):
     snap_list = ioc.IOCage(jail=jail).snap_list(_long, _sort)
 
     if header:
-        table.header(["NAME", "CREATED", "RSIZE", "USED"])
+        if jail == 'ALL':
+            cols = ["JAIL"]
+        else:
+            cols = []
+        cols.extend(["NAME", "CREATED", "RSIZE", "USED"])
```

Comment:
Might be a personal preference, but it would seem more concise to write:
```suggestion
        cols = ["NAME", "CREATED", "RSIZE", "USED"]
        if jail == 'ALL':
            cols.insert(0, "JAIL")
```
Or use `append("JAIL")` if the position doesn't matter (I assumed it did)
It seems more obvious this way that there are 4 default columns

Reply:
According to this PR #17 the `list.insert(0, item)` pattern comes with a performance penalty so it was refactored away. That's why I avoided it here too.

Reply:
That's fair, though it would probably be negligible here, let's not introduce it back :+1: